src/components/VehicleEditModal.jsx

```
@@ -0,0 +1,55 @@
+}) {
+  const [vehicleNumber, setVehicleNumber] = useState('');
+
+  if (!isOpen) return null;
+
+  return createPortal(
+    <div className="fixed inset-0 z-50 flex items-center justify-center bg-gray-400/50 backdrop-blur-sm">
+      <div className="w-full max-w-md rounded-2xl bg-white p-6 shadow-xl">
+        <h2 className="head2 mb-4 text-center">차량 번호 수정</h2>
+        <div className="mb-4">
+          <label className="text-sm text-gray-500">기존 차량 번호</label>
+          <div className="mt-1 w-full rounded-md border bg-gray-100 px-3 py-2 text-gray-700">
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)